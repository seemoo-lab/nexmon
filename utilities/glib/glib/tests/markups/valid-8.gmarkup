<foo><![CDATA[ some <<<<>>>> CDATA ]]></foo>
