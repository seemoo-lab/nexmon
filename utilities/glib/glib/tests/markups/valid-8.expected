ELEMENT 'foo'
  TEXT ''
  PASS '<![CDATA[ some <<<<>>>> CDATA ]]>'
  TEXT ''
END 'foo'
